ansible/roles/ocp-infra-osp-fip/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+key_algorithm: hmac-md5
```

Comment:
The var name is wrong. It should be `ddns_key_algorithm` to match what you are using in the task.

Reply:
Oops. Fixed.